omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderController1_10Test.java

```diff
@@ -589,4 +590,38 @@ public void invalidCareCenterShouldThrowException() throws Exception {
 		        );
 		handle(req);
 	}
+
+	@Test(expected = InvalidSearchException.class)
+	public void doSearch_shouldThrowExceptionIfNoPatientUuidIsSpecified() throws Exception {
+		MockHttpServletRequest req = newGetRequest(getURI(),
+				new Parameter("status", "active")
+		);
+		handle(req);
+	}
+
+	@Test
+	public void shouldGetOrdersByPatientUuid() throws Exception {
+
+		Patient patient = patientService.getPatientByUuid(PATIENT_UUID);
+
+		// get all drug orders
+		List<Order> activeDrugOrders = orderService.getActiveOrders(patient, orderService.getOrderTypeByName("Drug order"),
+				null, null);
+		int activeDrugOrderCount = activeDrugOrders.size();
+
+		// get all test orders
+		List<Order> activeTestOrders = orderService.getActiveOrders(patient, orderService.getOrderTypeByName("Test order"),
+				null, null);
+		int activeTestOrderCount = activeTestOrders.size();
+
+		// order service should return all active orders with patientUuid specified
+		MockHttpServletRequest req = newGetRequest(getURI(),
+				new Parameter("patient", PATIENT_UUID),
+				new Parameter("status", "active")
+		);
+		SimpleObject orders = deserialize(handle(req));
+		ArrayList<Object> resp = (ArrayList<Object>) PropertyUtils.getProperty(orders, "results");
+		assertEquals(activeTestOrderCount + activeDrugOrderCount, resp.size());
```

Comment:
@IamMujuziMoses we can use ` activeDrugOrders.size() + activeTestOrders.size()` without creating two variables. 
```suggestion
		assertEquals( activeDrugOrders.size() + activeTestOrders.size(), resp.size());
```